tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: Include tasks for sanity checks
   ansible.builtin.import_tasks: "sanity-check.yml"
 
+- name: Create Swarm Client user/group
+  ansible.builtin.import_tasks: "add-jenkins-user.yml"
+  when: swarm_client_create_user
```

Comment:
Line 17 includes `{{ ansible_os_family | lower }}/install.yml`, which includes `unix/add-jenkins-user.yml` or `windows/add-jenkins-user.yml`, so we always include these tasks.

Reply:
Good point. I've reworked this branch a bit and made a separate commit to include this task explicitly, see 31b2c7a.